app/src/ui/lib/sandboxed-markdown.tsx

```diff
@@ -105,8 +118,27 @@ export class SandboxedMarkdown extends React.PureComponent<
     this.frameContainingDivRef = frameContainingDivRef
   }
 
+  private initializeMarkdownEmitter = async () => {
```

Comment:
Does this need to be `async`? I don't see any `await` 🤔 
```suggestion
  private initializeMarkdownEmitter = () => {
```

Reply:
Thanks good catch -> I had moved stuff around and forgot to remove that.